common/scripts/ci/run_dashboard_plugin_tests.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+DIR=$(echo $(cd $(dirname "${BASH_SOURCE[0]}") && pwd -P))
+_RUSH="${DIR}/docker_rush.sh"
+_RUSHX="${DIR}/docker_rushx.sh"
+
+$_RUSH install
+$_RUSH build -t dashboard-plugin-tests
+$_RUSHX tools/dashboard-plugin-tests build
```

Comment:
Is this second build needed? It should already be built by the previous command, right? The lowercase `-t` builds the target package as well, as opposed to `-T` that does not build the target package.

Reply:
Removed